app/move/app/view/views/_layout.njk

```diff
@@ -30,11 +30,20 @@
 
   {% if moveIsLockout %}
     <div class="govuk-!-margin-bottom-6 prison-lockout-banner">
-      <div class="prison-lockout-banner-content">
-        {{ govukWarningText({
-          text: t("moves::is_lockout"),
-          iconFallbackText: "Warning"
-        }) }}
+      <div class="govuk-width-container prison-lockout-banner-content">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            {{ govukWarningText({
+              text: t("moves::is_lockout"),
+              iconFallbackText: "Warning"
+            }) }}
+          </div>
+          <div class="govuk-grid-column-one-third">
+            {% if moveLodgingStarted === true and moveLodgingEnded === false %}
```

Comment:
```suggestion
            {% if moveLodgingStarted and !moveLodgingEnded %}
```

Reply:
I don't think NJK supports the `!` operator on a variable

Reply:
NJK supports `not` instead of `!`, so maybe this should be `if moveLodgingStarted and not moveLodgingEnded`

Reply:
Oh nice!